### PR TITLE
add my hometown file :smile:

### LINKED ISSUE #20: mortoland checklist
- [x] Create an issue
- [x] Create branch
- [x] Create a Pull Request
- [x] Collaborate
- [x] Merge

### PR DESCRIPTION
**Pull request by mortoland**
Just adding a hometown file for Chicago, IL with my recommendation!!!!!!!!!!!!!!!!!!!!!!!!!!!!! :small_airplane:  
Closes #20

### CODE REVIEW

**Comment by cynthiaj13** [comment]
You didn't put any recommendations!! :disappointed: 

Can't approve until there is at least one recommendation :smile:

**Review by cynthiaj13 — APPROVED** [review]
No recommendations but I will approve this time :-1:

File: mortoland-chicago.IL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Chicago, IL
+
+
+## Places to Eat
```
Comment (cynthiaj13) [inline mortoland-chicago.IL:4]: You should add a list, with your first bullet being Marisa's on Thursdays!

**Comment by mortoland** [comment]
cool works for me !

File: mortoland-chicago.IL [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+## Things to Do
+- Planetarium
+- Museums
```
Comment (cynthiaj13) [inline mortoland-chicago.IL:9]: Which one's your favorite? I love the Field Museum :us:

File: mortoland-chicago.IL [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Chicago, IL
+
+
+## Places to Eat
+
+
+## Things to Do
+- Planetarium
```
Comment (cynthiaj13) [inline mortoland-chicago.IL:8]: Have you done Adler After Dark?? Every third Thursday at the Adler Planetarium is a 21+ event with lots of alcohol...and it's a different theme each time!!!

File: mortoland-chicago.IL (hunk 2 above)
Comment (mortoland) [reply mortoland-chicago.IL:9]: Yes Field Museum is cool for sure!

**Pull request merged**